lib/stacks/data.ts

```diff
@@ -0,0 +1,156 @@
+import * as cdk from "aws-cdk-lib";
+import * as dynamodb from "aws-cdk-lib/aws-dynamodb";
+import { Construct } from "constructs";
+import { DynamoDBTable } from "../local-constructs/dynamodb-table";
+import {
+  aws_iam as iam,
+  aws_lambda as lambda,
+  aws_lambda_nodejs as lambda_nodejs,
+} from "aws-cdk-lib";
+
+interface DatabaseStackProps extends cdk.NestedStackProps {
+  project: string;
+  stage: string;
+  stack: string;
+  isDev: boolean;
+}
+
+export class DatabaseStack extends cdk.NestedStack {
+  public readonly tables: { [name: string]: dynamodb.Table };
+  public readonly seedDataFunction: lambda_nodejs.NodejsFunction | undefined;
+
+  constructor(scope: Construct, id: string, props: DatabaseStackProps) {
+    super(scope, id, props);
+
+    const stage = this.node.tryGetContext("stage") || "dev";
+
+    this.tables = {
+      "age-ranges": new DynamoDBTable(this, "AgeRanges", {
```

Comment:
these are the same tables defined in data, yea? should this be a shared definition somewhere? or can api consume this?

Reply:
wait is this just a duplicate of the data stack? where is the api definitions?